tests/unit/test_base_test.py

```diff
@@ -0,0 +1,169 @@
+from unittest.mock import MagicMock, patch
+
+import pytest
+from selenium.common.exceptions import TimeoutException, ElementNotInteractableException
+
+from pytest_splunk_addon_ui_smartx.base_test import UccTester
+
+
+@pytest.fixture()
+def ucc_mock():
+    ucc = UccTester()
+    ucc.wait = MagicMock()
+
+    def f(x):
+        out = x("fake_browser")
+        if out:
+            return out
+        raise TimeoutException
+
+    ucc.wait.until.side_effect = f
+    return ucc
+
+
+@pytest.mark.parametrize(
+    "left, right, operator",
+    [
+        (lambda x: x, lambda x: x, "=="),
+        (lambda x: x, lambda x: not x, "!="),
+        (lambda x: -x, lambda x: x, "<"),
+        (lambda x: -x, lambda x: x, "<="),
+        (lambda x: x, lambda x: x, "<="),
+        (lambda x: x, lambda x: -x, ">"),
+        (lambda x: x, lambda x: -x, ">="),
+        (lambda x: x, lambda x: x, ">="),
+        (lambda x: x, lambda x: [x, 1, 2], "in"),
+        (lambda x: x, [], "not in"),
+        (lambda x: x, lambda x: x, "is"),
+        (lambda x: x, lambda x: not x, "is not"),
+    ],
+)
+def test_operator_map_callable_obj_pass(ucc_mock, left, right, operator):
+    args = {"x": 1}
+    assert ucc_mock.assert_util(left, right, operator, args, args) is None
+
+
+@pytest.mark.parametrize(
+    "left, right, operator",
+    [
+        (lambda x: x, lambda x: -x, "=="),
+        (lambda x: x, lambda x: x, "!="),
+        (lambda x: x, lambda x: x, "<"),
+        (lambda x: x, lambda x: -x, "<"),
+        (lambda x: x, lambda x: -x, "<="),
+        (lambda x: x, lambda x: x, ">"),
+        (lambda x: -x, lambda x: x, ">"),
+        (lambda x: -x, lambda x: x, ">="),
+        (lambda x: x, lambda x: [], "in"),
+        (lambda x: x, lambda x: [x, 1, 2], "not in"),
+        (lambda x: x, lambda x: not x, "is"),
+        (lambda x: x, lambda x: x, "is not"),
+    ],
+)
+def test_operator_map_callable_obj_assert_error(ucc_mock, left, right, operator):
+    args = {"x": 1}
+    with pytest.raises(AssertionError):
+        assert ucc_mock.assert_util(left, right, operator, args, args) is None
```

Comment:
Maybe it is enoguh?:
```suggestion
        ucc_mock.assert_util(left, right, operator, args, args)
```
`AssertionError` is raised inside function probably